chaoscenter/Makefile

```
@@ -97,15 +97,21 @@ backend-services-checks:
 	# 	&& exit 1; \
 	# fi
 
-unit-tests:
+backend-unit-tests:
 	@echo "------------------"
-	@echo "--> Running unit tests"
+	@echo "--> Running backend unit tests"
 	@echo "------------------"
-	@cd graphql-server && go test -cover ./...
-#	@cd authentication && go test -v ./...
-#	@cd cluster-agents/subscriber && go test -v ./...
+	@cd graphql/server && go test -cover ./...
```

```suggestion
	@cd graphql/server && go test -v ./...
```

For now coverage is not required!